packages/components/src/components/icon/common/ic-dashboard-component-tab.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is it possible to minify this file as well?

Reply:
done